content/en/tutorials/gitlab_ci_testcontainers/index.md

```diff
@@ -0,0 +1,431 @@
+---
+title: "End-to-End Testing in Gitlab CI with Testcontainers and LocalStack: Understanding Runners and Docker in Docker"
```

Comment:
I would drop the last part of the title to keep it simple & crisp. It should be okay to have this info in the description

Reply:
This is done on purpose, as the main pain points of running on GitLab are the runners and the docker in docker setup. Additionally, all tutorial titles are pretty long, so it's not sticking out from an aesthetic point of view.